Applications/Calendar/CalendarWidget.h

```diff
@@ -0,0 +1,47 @@
+#pragma once
+
+#include "Calendar.h"
+#include <LibGUI/Frame.h>
+#include <LibGUI/Label.h>
+#include <LibGUI/Widget.h>
+
+class CalendarWidget final : public GUI::Widget {
+    C_OBJECT(CalendarWidget)
+
+public:
+    CalendarWidget();
+    virtual ~CalendarWidget() override;
+
+private:
+    void update_calendar_tiles(int target_month, int target_year);
+    void show_add_event_window(Calendar* calendar);
+
+    //TODO: Change to RefPtr or NonNullRefPtr...?
+    Calendar* m_calendar;
```

Comment:
For a single-owner object, `OwnPtr<Calendar>`.
For a multiple-owner object, `RefPtr<Calendar>`.
My understanding here is that `CalendarWidget` singly owns both `Calendar` and all the `CalendarTile` objects(?)

Reply:
I think calendar tile objects are shared with window widget which is why the RefPtr is used for `RefPtr<CalendarTile> m_calendar_tiles[35]`.